core/field_variable.js

```diff
@@ -48,7 +48,7 @@ goog.require('goog.string');
  */
 Blockly.FieldVariable = function(varname, opt_validator, opt_variableTypes) {
   Blockly.FieldVariable.superClass_.constructor.call(this,
-      Blockly.FieldVariable.dropdownCreate, opt_validator);
+    Blockly.FieldVariable.dropdownCreate, opt_validator);
```

Comment:
nit, undo the indent edits.

Reply:
OK.
but why my editor make the indent like that ?
I use VS Code , 
maybe we need to edit the eslintrc to make it better ?

Reply:
I haven't used VS Code before, but if it's using eslint you can use Google's configuration: https://github.com/google/eslint-config-google